src/visibility.js

```diff
@@ -118,13 +125,20 @@ class Visibility extends BasePlugin {
   _stopFloating() {
     Utils.Dom.removeClassName(this._floatingContainer, FLOATING_ACTIVE_CLASS);
     Utils.Dom.removeAttribute(this._floatingContainer, 'style');
+    if (this.config.floating.draggable) {
+      this.eventManager.unlisten(this._floatingContainer, 'mousedown');
+      this._dragStop();
+    }
   }
 
   _startFloating() {
     Utils.Dom.addClassName(this._floatingContainer, FLOATING_ACTIVE_CLASS);
     Utils.Dom.setStyle(this._floatingContainer, 'height', this.config.floating.height + 'px');
     Utils.Dom.setStyle(this._floatingContainer, 'width', this.config.floating.width + 'px');
     Utils.Dom.setStyle(this._floatingContainer, 'margin', `${this.config.floating.marginY}px ${this.config.floating.marginX}px`);
+    if (this.config.floating.draggable) {
+      this.eventManager.listen(this._floatingContainer, 'mousedown', this._dragMouseDown.bind(this));
```

Comment:
don't you need to remove [this](https://github.com/kaltura/playkit-js-visibility/pull/9/files#diff-9c3f2b305c531447d257c17efb63ef3cR200) and [this](https://github.com/kaltura/playkit-js-visibility/pull/9/files#diff-9c3f2b305c531447d257c17efb63ef3cR201) listeners on `mouseup` of `_floatingContainer`?
you keep listening to changes in document events when you don't need to.